editor/plugins/script_text_editor.cpp

```diff
@@ -239,6 +239,29 @@ void ScriptTextEditor::_show_warnings_panel(bool p_show) {
 void ScriptTextEditor::_warning_clicked(Variant p_line) {
 	if (p_line.get_type() == Variant::INT) {
 		goto_line_centered(p_line.operator int64_t());
+	} else if (p_line.get_type() == Variant::DICTIONARY) {
+		Dictionary meta = p_line.operator Dictionary();
+		const int line = meta["line"].operator int64_t() - 1;
+
+		CodeEdit *text_editor = code_editor->get_text_editor();
+		String prev_line = text_editor->get_line(line - 1);
+		if (prev_line.contains("@warning_ignore")) {
+			const int closing_bracket_idx = prev_line.rfind(")");
```

Comment:
Not sure if this is a common use case, but if there's a comment at the end of like containing a `)` this will break.